docs/tools/bootloader.md

```diff
@@ -0,0 +1,48 @@
+## Boot loader Configuration
+
+This page describes each boot loader configuration parameter in detail.
+For an guide on using the managed and unmanaged boot loader modes, see [the boot loader turorial](../tutorials/bootloder.md).
+
+There are 4 configuration parameters that affect the location of an application in ROM:
+ * `target.mbed_app_start`
+ * `target.mbed_app_size`
+ * `target.bootloader_img`
+ * `target.restrict_size`
+ 
+All of these parameters are valid in `targets.json`, `mbed_lib.json` and `mbed_app.json` and may be defined for individual targets and the wildcard target, `*`. A parameter in `mbed_lib.json` overrides a parameter in `targets.json`, and a parameter in `mbed_app.json` overrides one in `mbed_app.json`.
+
```

Comment:
Is this intended? I assume one should be `mbed_lib.json`.

Reply:
Ha! Good catch. Yes, the second `mbed_app.json` should be `mbed_lib.json` in the last sentence.

Reply:
Fixed with a rebase.